etcd-client/Cargo.toml

```diff
@@ -18,16 +18,16 @@ async-std = "1.8.0"
 async-trait = "0.1"
 backoff = { version = "0.3.0", features = ["futures"] }
 bytes = "0.5"
+clippy-utilities = "0.1.0"
 futures = "0.3.5"
 grpcio = { version = "0.9.1", default-features = false, features = ["protobuf-codec"] }
 http = "0.2"
 log = "0.4.11"
-lockfree-cuckoohash = { git = "https://github.com/datenlord/lockfree-cuckoohash.git", rev = "69ccd5a"}
+lockfree-cuckoohash = { git = "https://github.com/datenlord/lockfree-cuckoohash", rev = "c2aff70"}
```

Comment:
we have published a cuckoohash

Reply:
etcd-client depends on a new feature of cuckoohash. Will update after that feature is published.